turn-io-adapter/src/main/java/org/egov/rb/contract/MessageRequest.java

```diff
@@ -0,0 +1,32 @@
+package org.egov.rb.contract;
+
+import java.util.List;
+
+import org.egov.common.contract.request.RequestInfo;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class MessageRequest {
+	
+	@JsonProperty("RequestInfo")
+	  private RequestInfo requestInfo = null;
+ 
+	@Autowired
```

Comment:
Please add not null validation using annotations on fields that are mandatory to create request. Add it in all POJO's wherever applicable

Reply:
In RequestInfo alreadynot null  is there and the remaining pojo classes we cannot add not  null those are for receiving data format

Reply:
Why can't we add?

Reply:
Added not null validation in pgr requestInfo and pojo class